Cargo.toml

```diff
@@ -1,4 +1,7 @@
 [workspace]
 members = [
     "rust",
-]
\ No newline at end of file
+]
+
+[patch.crates-io]
+chrono = { git = "https://github.com/sinkingsugar/chrono" }
```

Comment:
Shouldn't we have the fork in the chainblocks org?

Reply:
indeed, this was an experiment that worked. nice catch.